modules/lvms-creating-volume-clones.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-volume-clones_{context}"]
+= Creating volume clones
+
+To create a clone of a persistent volume claim (PVC), you must create a `PersistentVolumeClaim` custom resource (CR) in the namespace where you created the source PVC.
+
+[IMPORTANT]
+====
+The cloned PVC has write access.
+====
+
+.Prerequisites
+
+* You ensured that the PVC is in `Bound` state. This is required for a consistent snapshot.
+
+.Procedure
+
+. Log in to the OpenShift CLI (`oc`).
++
+[IMPORTANT]
+====
+If you are using {lvms} in {rh-rhacm}, log in to the {rh-rhacm} CLI using your {product-title} credentials.
+====
+
+. Create a `PersistentVolumeClaim` custom resource (CR) YAML file.
++
+.Example `PersistentVolumeClaim` CR to create a volume clone
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+Metadata:
+  name: lvm-block-1-clone
+Spec:
+  storageClassName: lvms-vg1 <1>
+    name: lvm-block-1
+    kind: PersistentVolumeClaim
+  accessModes:
+   - ReadWriteOnce 
+  volumeMode: Block <2>
+  Resources:
+    Requests:
+      storage: 2Gi <3>
+----
+<1> Set this field to the value of the `storageClassName` field in the `PersistentVolumeClaim` CR of the source PVC.
+<2> To create a block PVC, set this field to `Block`. To create a file PVC, set this field to `Filesystem`.
+<3> Specify the storage for the cloned PVC. A PVC cannot be cloned with a different size.
```

Comment:
```suggestion
<3> Specify the storage size for the cloned PVC. A PVC cannot be cloned with a different size.
```

Reply:
@radeore - Is the following sentence correct?
"The storage size of the cloned PVC must be greater than or equal to the storage size of the source PVC."

When we use "must be" - then it implies that the storage size of the cloned PVC cannot be lesser than the storage size of the source PVC. Is this true?